docs/esi-leap-requirements.md

```diff
@@ -0,0 +1,15 @@
+# MOC esi-leap Requirements
+
+## Overview
+esi-leap is the leasing part of [Elastic Secure Infrastructure](https://github.com/CCI-MOC/esi) (esi), an OpenStack service for acting as a FLOCX provider to communicate with a FLOCX marketplace. It is intended to work on top of the [esi-common library](https://github.com/CCI-MOC/esi-common), which supports an OpenStack install that simulates Ironic multi-tenancy through the use of `project_owner_id` and `project_id` property attributes.
+
+## Definitions
+
+### Offer
+Baremetal node owners can offer up their nodes to others with nodes configurations, leasing period, and the price. On behalf of the node owners, esi-leap sends the offers to a FLOCX marketplace. An offer can be created, destroyed, saved, retrieved, and expired by esi-leap.
```

Comment:
It is not supposed to automatically send offers to FLOCX marketplace. I currently have a pending pr to fix this issue.

Reply:
But we could mention (perhaps in the overview?) that a marketplace service could be built on top of esi-leap. But regardless I feel like introducing the marketplace service here could be confusing

Reply:
Yes, FLOCX could be confusing here. If your pr merges, I can make changes according to the newest repo.

Reply:
The remove-flocx PR has merged, so feel free to remove any references here!

Reply:
As this a requirements doc, it may also make sense to explicitly list offer fields (and the same with contract below).